ndk/src/asset.rs

```diff
@@ -14,7 +14,7 @@ use std::{
 /// A native [`AAssetManager *`]
 ///
 /// [`AAssetManager *`]: https://developer.android.com/ndk/reference/group/asset#aassetmanager
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq, Hash)]
 #[doc(alias = "AAssetManager")]
 pub struct AssetManager {
```

Comment:
Can be `Clone`/`Copy` since this is a cheap reference/handle from `Activity` (but should carry a lifetime on `Activity` realistically).